README.md

```diff
@@ -3,15 +3,84 @@
 This repository contains the REST API running as a PubNub function that is required for
 the [ChatEngine Framework](https://github.com/pubnub/chat-engine/) to operate.
 
-## Setup
+## ChatEngine Application and Key Provisioning Guide
 
-First, configure your application in the PubNub administration UI.
+To setup ChatEngine on PubNub, one must first setup a new PubNub App. The following outlines how to manually setup a PubNub ChatEngine Key.
 
-[How to Setup your account for ChatEngine](guides/ChatEngine_Application_and_Key_Provisioning.md)
+Setup an account, create a set up pubsub keys, configure these keys to 
```

Comment:
Incomplete sentence. Also, you "set up" an account, but you deliver the "setup" for a joke.

Reply:
"pub/sub keys"